.drone.star

```diff
@@ -5,11 +5,30 @@
 # with a specific compiler.
 #
 
-OC_TESTING_MIDDLEWARE = "owncloud/owncloud-test-middleware:1.4.0"
+CYTOPIA_BLACK = "cytopia/black"
+DOCKER_GIT = "docker:git"
 GUI_TEST_REPORT_DIR = "/drone/src/test/guiReportUpload"
 GUI_TEST_DIR = "/drone/src/test/gui"
 NOTIFICATION_TEMPLATE_DIR = "/drone/src"
 
+MYSQL = "mysql:8.0"
+OC_CI_ALPINE = "owncloudci/alpine:latest"
+OC_CI_BAZEL_BUILDIFIER = "owncloudci/bazel-buildifier"
+OC_CI_CLIENT = "owncloudci/client:2.9-amd64"
+OC_CI_CORE = "owncloudci/core"
+OC_CI_DRONE_CANCEL_PREVIOUS_BUILDS = "owncloudci/drone-cancel-previous-builds"
+OC_CI_PHP = "owncloudci/php:7.4"
+OC_CI_SQUISH = "owncloudci/squish:qt512"
+OC_CI_TRANSIFEX = "owncloudci/transifex:latest"
+OC_TESTING_MIDDLEWARE = "owncloud/owncloud-test-middleware:1.4.0"
+OC_UBUNTU = "owncloud/ubuntu:20.04"
+PLUGINS_GIT = "plugins/git-action:1"
```

Comment:
const name PLUGINS_GIT_ACTION would be better because we have used names for all the apps excluding the tag name only.